rwengine/src/script/modules/GameModule.cpp

```diff
@@ -1385,7 +1385,7 @@ GameModule::GameModule()
 	bindUnimplemented( 0x03DA, game_set_garage_follow_player, 1, "Set Garage Camera Follows Player" );
 	
 	bindFunction(0x03DC, game_add_object_blip<PickupObject>, 2, "Add blip for pickup");
-	bindUnimplemented(0x03DD, game_add_object_sprite_blip<PickupObject>, 3, "Add Sprite Blip for Pickup"); /// @todo crashes LUIGI4 because of missing 032B
+	bindFunction(0x03DD, game_add_object_sprite_blip<PickupObject>, 3, "Add Sprite Blip for Pickup");
```

Comment:
At the time of writing the commit is called: "0x3DD: Re-enable as 0x32B is now implemented"
This could mean various things. It's really ambiguous. It could be mistaken for flag values or version numbers of some file format.
Something like "Script: enable 0x03DD because it depended on recently fixed 0x32B" is more descriptive. Other buzz-words such as "opcode" "SCM" etc would have worked too (note that I also highlighted the dependency).